main.js

```diff
@@ -0,0 +1,49 @@
+import * as ort from "https://cdn.jsdelivr.net/npm/onnxruntime-web/dist/esm/ort.min.js";
+ort.env.wasm.wasmPaths = "https://cdn.jsdelivr.net/npm/onnxruntime-web/dist/";
+async function main() {
+    try {
+        const runButton = document.getElementById('runInference');
+        runButton.addEventListener('click', async () => {
+            const modelCardData = document.getElementById('modelcard').value;
```

Comment:
why not use the jquery you loaded in index.html to get the elements?

Reply:
You are right, this should be refactored (code was written before I decided on loading jquery)